### PR TITLE
Move switch large video logic to react

### PR DESCRIPTION
**Pull request by hristoterezov**
Now switching the onstage participant and sending the select participant event should be always tied together.

### CODE REVIEW

File: react/features/large-video/subscriber.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+// @flow
+
+import { StateListenerRegistry } from '../base/redux';
+import VideoLayout from '../../../modules/UI/videolayout/VideoLayout';
+
+/**
+ * For auto-pin mode, listen for changes to the known media tracks and look
```
Comment (virtuacoplenny) [inline react/features/large-video/subscriber.js]: Probably want to change doc this but it still kind of makes sense.

Comment (hristoterezov) [reply react/features/large-video/subscriber.js]: Yep. Copy/paste error 🤦‍♂ 

It is fixed now!

File: react/features/large-video/subscriber.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+import { StateListenerRegistry } from '../base/redux';
+import VideoLayout from '../../../modules/UI/videolayout/VideoLayout';
+
+/**
+ * Updates the on stage participant video.
+ */
+StateListenerRegistry.register(
+    /* selector */ state => state['features/large-video'].participantId,
+    /* listener */ participantId => {
+        VideoLayout.updateLargeVideo(participantId, true);
```
Comment (saghul) [inline react/features/large-video/subscriber.js]: This change just broke mobile. `VideoLayout` is not working there.

Comment (hristoterezov) [reply react/features/large-video/subscriber.js]: Ouchh... Sorry! Will fix it!

**Pull request merged**